ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -178,7 +178,7 @@ define('setup_manage_taxonomies', ['reactaddons', 'lodash', 'jquery', 'utils'],
       }).fail(function(data) {
         var jsonData = data.responseJSON;
         var i = 0;
-        if (jsonData.non_field_errors &&
+        if (jsonData && jsonData.non_field_errors &&
           jsonData.non_field_errors.length > 0) {
```

Comment:
You can remove this check, jsonData.non_field_errors.length is checked within the for loop